### PR TITLE
fix: swap operation order enum

### PR DESCRIPTION
**Pull request by mrbasado**

### CODE REVIEW

File: src/interface/IWrappedETH.sol [hunk 1]
```diff
@@ -6,5 +6,5 @@ pragma solidity 0.8.17;
 /// @notice Wrapped ETH (version 9)
 /// @dev check https://etherscan.io/token/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#code
 interface IWrappedETH {
-    function withdraw(uint wad) external;
+    function withdraw(uint256 wad) external;
```
Comment (mrbasado) [inline src/interface/IWrappedETH.sol:9]: not sure why linter added this

Comment (dapp-whisperer) [reply src/interface/IWrappedETH.sol:9]: it's considered correct to not use the shorthand `uint`

**Comment by dapp-whisperer** [comment]
## Review
- Adjusts `EthVariantZapOperationType` action type ordering to match core protocol.
- Remainder of changes are linting to use prettier (default solidity settings)

**Comment by mrbasado** [comment]
can package in something else

**Pull request closed without merging**